portfolio/src/main/webapp/script.js

```diff
@@ -13,19 +13,70 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
```

Comment:
Is 'Adds' accurate? As far as I can tell the fact is replaced.

Reply:
I changed "adds" to "displays"